tests/e2e/pageObjects/FilesPage.js

```diff
@@ -1,17 +1,20 @@
 const Ocis = require('./OcisPage')
 const config = require('../config')
+const util = require('util')
 
 class Files {
   constructor() {
-    this.contextMenuBtnSelector = '.resource-table-btn-action-dropdown'
+    this.contextMenuBtnSelector = '.resource-%s-btn-action-dropdown'
```

Comment:
we should be able to use this single pattern to match both
```suggestion
    this.contextMenuBtnSelector = '[class*=-btn-action-dropdown]'
```